src/components/nodes/values/Image.vue

```diff
@@ -1,17 +1,34 @@
 <template>
   <div >
     <img v-if="image" :src="image.url" width="100%" />
-    <div class="q-px-md q-pt-md">
+    <div class="q-pt-md image-input-style text-center">
+      <b>Drop image here</b>
```

Comment:
This should probably say "Click or drag and drop file here", to be more descriptive of its entire functionality.